configure.ac

```diff
@@ -86,7 +86,7 @@ util.h \
 ])
 
 # Checks for optional header files with prerequisites of other headers.
-AC_CHECK_HEADERS([net/if_var.h],[],[],[#include <net/if.h>])
```

Comment:
After #701, `net/if_var.h` is not needed any more . You may remove this paragraph altogether.

Reply:
Ok, I will rebase today, thanks.

Reply:
I solved the conflict and merged from the web interface. Thank you very much for this improvement of portability @barracuda156.

Reply:
Thank you! (Sorry, I got distracted by other stuff.)